package.json

```diff
@@ -18,9 +18,10 @@
   },
   "scripts": {
     "clean": "rm -r dist",
-    "lint": "eslint . && prettier . --check && npm run rule-syntax-check",
+    "lint": "eslint . && prettier . --check && npm run validate-rules",
```

Comment:
Should "lint" call "generate-rule-schema" all the time, so it's updated whenever you update the TS type? Note that lint check is run on every PR

Reply:
👍 good idea. 63951c8